src/util.js

```diff
@@ -14,7 +14,7 @@
  */
 var Util = window.Util || {};
 
-Util.MIN_TIMESTEP = 0.001;
+Util.MIN_TIMESTEP = 0.0001;
```

Comment:
Very odd. Seems from your CL like FF gives timestamps in microseconds, but you convert to milliseconds. It seems impossible that your update rate is faster than the new timestep (which is 1/0.0001=10,000 Hz), since phone firmware only gives sensors at 200 Hz at most.

Could you investigate?